index.ts

```diff
@@ -18,14 +18,15 @@ const api = <T>(route: string, options?: Options): Promise<T> =>
     body: options?.body ? JSON.stringify(options.body) : undefined,
     headers: {
       ...options?.headers,
+      ["content-type"]: "application/json",
```

Comment:
This was the big issue. Without it, the POST response wasn't being interpreted correctly.

Reply:
The response received from the server? This header should only affect the request body but there might be something wrong somewhere

Reply:
Yeah, sorry, I phrased that wrong. The _request_ wasn't interrupted correctly.